### PR TITLE
156794815 help link button

### PR DESCRIPTION
**Pull request by rckopteri**
# Added
* Link to sea route editor user guide

### CODE REVIEW

File: ote/src/cljs/ote/ui/buttons.cljs [hunk 1]
```diff
@@ -27,3 +29,11 @@
                     :font-size "12px"
                     :font-weight "bold"}} opts)
     label]])
+
+(defn open-link [e! url]
```
Comment (tatut) [inline ote/src/cljs/ote/ui/buttons.cljs]: could we use the common `linkify` utility instead of this new button?

Comment (K1ll3rF0x) [reply ote/src/cljs/ote/ui/buttons.cljs]: Yes, please.

Comment (rckopteri) [reply ote/src/cljs/ote/ui/buttons.cljs]: We could, but ui spec says that it must be a flat-button

Comment (K1ll3rF0x) [reply ote/src/cljs/ote/ui/buttons.cljs]: Well, you could do it like this:
```cljs
[ui/flat-button
  {:label (r/as-element 
               (common/linkify "http://nonsense" "Go!" {:target "_blank"}))
  :primary true}]
```

Comment (rckopteri) [reply ote/src/cljs/ote/ui/buttons.cljs]: That won't work. You can press the button and nothing happens if mouse pointer is not on top of label.

Comment (K1ll3rF0x) [reply ote/src/cljs/ote/ui/buttons.cljs]: It turns out, you can actually do it like this:
```cljs
 (common/linkify "http://nonsense" 
  [ui/flat-button {:label "Go! :primary true}]
   {:target "_blank"})
```
The a-element will properly wrap the "button" element and the whole button is then interpreted as a link.

Comment (rckopteri) [reply ote/src/cljs/ote/ui/buttons.cljs]: This is why I love these reviews. Thanx! That linkify works marvellously.
Changes coming up.

**Pull request merged**